htdocs/public/users/view.php

```diff
@@ -341,11 +341,15 @@
 }
 
 // Social networks
+if (empty($object->socialnetworks)) {
```

Comment:
if $object->socialnetworks is empty then getSocialNetworks($object->id) will return empty too.
Because $object->socialnetworks is loaded by the fetch of user the same way the getSocialNetworks do it.